articles/libraries/lock/index.md

```diff
@@ -120,21 +120,13 @@ This is just one example of how **Lock 10** could work with a **Single Page Appl
 
 The [API reference][lock-api] provides more comprehensive documentation of the API and its various available methods, and the [Customization reference][lock-customization] details out the different options that you have for customizing Lock to meet your specific needs.
 
-## Examples
-
-The **example** directory has a ready-to-go app. In order to run it you need [node](http://nodejs.org/) installed.
-
-Then execute `npm i` to install dependencies (only once) and `npm example` from the root of this project.
-
-Finally, point your browser at `http://localhost:3000/` and play around.
-
-## Start using Lock!
+## Using auth0.js
 
-${lockSDK}
+There is no need to use `getClient()` any more - instead, you can just instantiate a new `Auth0` object.
```

Comment:
- Using "need" suggests that `lock.getClient()` is still available but it is not.
- The reader probably doesn't know what is an `Auth0` object, what is auth0.js, or why does she needs to use them, specially if this is the first time is using lock.